### ISSUE TITLE
Wrong localisation for "main" in french

### ISSUE THREAD

**Issue by m4dz** [cozy/cozy-contacts#218]
Email datapoints labels are wrong in fr: _main_ is translated _internet_, which seems to be confusing.

**Comment by poupotte** [comment]
I don't reproduce on my cozy for the moment.

*internet* doesn't correspond to a translation in transifex (https://www.transifex.com/cozy/cozy-contacts/viewstrings/#fr/client?qt=internet).

**Comment by ArchangeGabriel** [comment]
Hum, it seems to be /main/ now. Still not good, but I can’t find it in transifex.

**Comment by clochix** [comment]
The datapoints types are defined in https://github.com/cozy/cozy-contacts/blob/master/client/app/config.coffee#L7 and doesn't appear that we can translate them for now.
@m4dz are this keys just missing in the english local file, or don't we have a mechanism to translate them?

**Comment by m4dz** [comment]
The case for datapoints labels is a little bit tricky : you can use whatever you want as a label for a datapoint (thanks to the VCard RFC) but you also be encouraged to use some default labels, like _MAIN_, _WORK_ or _HOME_ (thanks again to the RFC). My thoughts is to give to the user a list of standard labels, which can be translated, but let him customize them if he wants (and then sync custom labels as-it).

It's not really complicated, but it needs a combo-box to do it (a select / suggestion of localized labels + the ability to input what you want). As long as HTML doesn't provides us a standard combo (thanks to the HTML Working Group and the so 90's form support in our browsers :rage:) we need to use Web Components to do it properly, and I hadn't much more time to do it yet (but it's on the map :smile:)

**Comment by aenario** [comment]
How about using tagit which we already use for tags ?

**Comment by m4dz** [comment]
> How about using tagit which we already use for tags ?

Not sure, but it can be a nice option, Let's take a look at it :smile:.

**Comment by Phyks** [comment]
Should a user use "main" rather than say "principal" in French? It is really misleading that every other field names are translated except "main".

**Comment by m4dz** [comment]
Yep @Phyks, this feature is planned to be developed on this next milestone (upcoming 3 months) on our roadmap :wink:.

**Comment by Phyks** [comment]
OK, thanks for the feedback !
